src/braket/default_simulator/openqasm/parser/openqasm_ast.py

```diff
@@ -0,0 +1,1059 @@
+# Apache V2 usage with no significant modifications
```

Comment:
If possible, can you put this, as well as the other files copied from OpenQASM to a dedicated folder, so they're easier to track (and remove when the time comes)

Reply:
That is this folder (everything that doesn't start with "braket"). I can further subdivide if you think that'd be cleaner, but I wanted to avoid `openqasm/parser/openqasm` along the path lol

Reply:
I was just thinking openqasm/parser/3p or something, but no strong opinion